wire-ios-data-model/Tests/Source/Utils/InvalidClientsRemovalTests.swift

```diff
@@ -117,6 +117,7 @@ class InvalidClientsRemovalTests: DiskDatabaseTest {
             syncMOC.zm_cryptKeyStore.encryptionContext.perform {
                 XCTAssertTrue($0.hasSession(for: clientId))
             }
+            // TODO: Ask why we check these - how is it linked to fingerpint?
             XCTAssertTrue(otherClient.hasSessionWithSelfClient)
             XCTAssertFalse(otherClient.isZombieObject)
```

Comment:
these two assertions are failing but I don't get how it is related to my changes (fingerprint)?

Reply:
they shouldn't be but if `XCTAssertFalse(otherClient.isZombieObject)` is failing then I would also expect `otherClient.hasSessionWithSelfClient` to fail.

Now `otherClient` is deleted and before it wasn't?

Reply:
removed as it does not make sense since we have the duplicate clients fix